user_guide_src/source/database/index.rst

```diff
@@ -14,10 +14,10 @@ patterns. The database functions offer clear, simple syntax.
     Connecting to a Database <connecting>
     Running Queries <queries>
     Generating Query Results <results>
-    Query Helper Functions <helpers>
+    Query Helper Methods <helpers>
```

Comment:
If the string is the same as the page title, why don't you remove it?
```suggestion
    helpers
```

Reply:
@kenjis To supply consistency between the menu on the left and the content header, I changed "Functions" to "Methods".

Reply:
Yes, that is no problem. 

I pointed out that if the string in the menu and the content header are the exactly same,
we don't need to write it in `index.rst`. If we don't write the string in `index.rst`, the inconsistency like this is never happens.

In other words, we only need to write it only when they are different.

Reply:
@kenjis I am just learning how the documentation works. I am reading the user guide from top to bottom (unlike a normal dev). When I see something erroneous (or unreasonable, or inconsistent or whatever it can be named) according to my mind, trying to offer  a correction. Until Working with Databases section, menu and the page headers were the same. Then I thought, in this section it was a typo and offered PR's. Of course menu and the headers can be different, of course menu can be shorter, but there are longer items in the menu (like "Database Manipulation with Database Forge" in the menu and Database Forge Class in the header. When I come to this part, I will offer to shorten the menu to "Database Manipulation" or "Database Forge").
Another reason with the "Query Helper Functions" was that, as far as I know a function in a class is called "method". If a function is not in any class it is called "function". I checked, the functions explained in this page are all methods of classes. That is why page header is correct saying "Query Helper Methods".

Reply:
Your corrections are no problem. 
We use *method* if it is in a class.

But you might not get what I mean.
We don't need to write "Query Helper Methods" in the `index.rst` if the page header is "Query Helper Methods".
See https://raw.githubusercontent.com/codeigniter4/CodeIgniter4/develop/user_guide_src/source/installation/upgrading.rst